src/p11_child/p11_child.h

```diff
@@ -59,7 +60,8 @@ enum pin_mode {
 };
 
 errno_t init_p11_ctx(TALLOC_CTX *mem_ctx, const char *ca_db,
-                     bool wait_for_card, struct p11_ctx **p11_ctx);
+                     bool wait_for_card, time_t timeout,
```

Comment:
Any reason to use `time_t` specifically and not plain `int` or `unsigned`?
This arg is just a period (number of seconds?), not a time point.

Reply:
Also code parses it as `POPT_ARG_LONG`, it's better to match types strictly to avoid surprises (especially on non-x86 platforms).

Reply:
for parsing I use long int but the rest of the code expects int. I did not have particular reason for using time_t I can change that to int and add check that this parameter fits into it.

Reply:
That's a rabbit hole. Looks like there is no portable way to increment time_t directly.